guide/getting-started/technology-primer.md

```diff
@@ -73,264 +79,265 @@ It's both. Lowercase bitcoin refers to the currency. So you may have 1 bitcoin i
    layout = "float-right-desktop"
 %}
 
-Bitcoin (as in "I have 1 bitcoin") are just numbers in the ledger of transactions often referred to as the bitcoin blockchain.
-
-A transaction involves sending bitcoin from one address to another. This reduces the available balance of the sending address and increases the balance of the receiving address. A small amount is deducted to pay for the transaction fee.
-
-Owning a bitcoin, therefore, means being in control of one or more addresses that have received bitcoin.
-
-</div>
-
-## How do I get an address?
+Bitcoin, as in "I have 1 bitcoin," are just numbers in a ledger of transactions referred to as the Bitcoin blockchain. Transactions assign bitcoin from one address to another. This process reduces the available balance of the sending address and increases the balance of the receiving address. A small amount is used as the transaction fee to pay for the verification.
 
-<div class="center" markdown="1">
-
-{% include image.html
-   image = "/assets/images/guide/getting-started/technology/addresses.jpg"
-   retina = "/assets/images/guide/getting-started/technology/addresses@2x.jpg"
-   alt-text = "Examples of addresses in a bitcoin wallet"
-   width = 400
-   height = 400
-   layout = "float-right-desktop"
-%}
+Therefore, owning a bitcoin means being in control of one or more addresses that have received bitcoin.
 
-Bitcoin wallet software can generate as many addresses as you need. Note that addresses are generated on your own computer. They follow a specific format, but there is no centralized system to verify or approve them. Valid addresses can even be generated on computers that have never been online.
+**More info**
```

Comment:
@danielnordh is using a different, sentence-based format for further reading. [For example](https://deploy-preview-501--sad-borg-390916.netlify.app/guide/designing-products/common-user-flows/#securing-a-wallet):

`See also, the Private key management section and Protecting a wallet.`

It's more compact and causes less of a visual break than the format you use with the bold "More info" and a single bullet list. How about we pick up Daniel's format? If there are 5+ links, I can imagine a bullet list work well, but for 1-2 links, the sentence works better, IMHO.